src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -21,24 +24,33 @@ public static final class Drivetrain {
 	
 	public static final class Arm {
 		//Motor port
-		public static final int MOTOR_PORT1 = 7;
-		public final static int MOTOR_PORT2 = 8;
+		public static final int LEFT_MOTOR_PORT = 7;
+		public final static int RIGHT_MOTOR_PORT = 8;
 		//all angles in rot here
-		public static final double intakeAngle = -.1;
-		public static final double ampAngle = .3;
-		public static final double speakerAngle = .4;
+		//TODO: finish understand why this is broken public static final Measure<Angle> INTAKE_ANGLE = Degrees.to(-1);
+		public static final double intakeAngle = Math.toDegrees(-0.1);
```

Comment:
these seem off. Please get real values from design, assuming resting angle is 0.